static/src/stylesheets/module/content/_photo-essay.scss

```diff
@@ -0,0 +1,93 @@
+.content--article {
+    .element-image--photo-esssay {
+        padding-bottom: 0;
+        margin-bottom: 0.5rem;
+
+        &.fig--no-caption.fig--has-shares {
+            padding-bottom: 0px !important;
+        }
+
+        .block-share {
+            display: none !important;
+        }
+        .caption {
+            max-width: 100%;
+            color: $news-main-1;
+        }
+        .article__fullscreen {
+            background-color: transparent;
+            border:1px solid rgba($neutral-6,0.5);
```

Comment:
Does this pass the linter? I guess there should be a space between `: 1px` and after the comma.